others/postfix_evaluation.cpp

```diff
@@ -0,0 +1,178 @@
+/**
+ * @file
+ * @brief Evaluation of Postfix Expression
```

Comment:
```suggestion
 * @brief Evaluation of [Postfix Expression](https://en.wikipedia.org/wiki/Reverse_Polish_notation)
```
Providing a reference link to the problem definition.